### PR TITLE
mobile: android polish omnibus

### PR DESCRIPTION
**Pull request by latter-bolden**
- fixes missing header and chat list section titles
- fixes unread counts not displaying
- fixes message actions crashing the app
- updates overlay to avoid attempting blur on Android
- avoids using iOS style action list on Android

Also added a fixture for the message actions (light & dark mode).

Fixes LAND-1793
Fixes LAND-1794
Fixes LAND-1806

### CODE REVIEW

File: apps/tlon-mobile/src/screens/ChannelScreen.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import type * as db from '@tloncorp/shared/dist/db';
 import * as store from '@tloncorp/shared/dist/store';
 import { handleImagePicked, useUploader } from '@tloncorp/shared/dist/store';
 import type { Story } from '@tloncorp/shared/dist/urbit';
-import { Channel, ChannelSwitcherSheet, View } from '@tloncorp/ui';
+import { Channel, ChannelSwitcherSheet, View } from '@tloncorp/ui/src';
```
Comment (dnbrwstr) [inline apps/tlon-mobile/src/screens/ChannelScreen.tsx:9]: src

Comment (dnbrwstr) [reply apps/tlon-mobile/src/screens/ChannelScreen.tsx:9]: Oops didn't mean to press add single comment lol.

Comment (latter-bolden) [reply apps/tlon-mobile/src/screens/ChannelScreen.tsx]: OUT:  _oops i forgot to undo replacing showDevTools with false_
IN: _oops i left in /src_

**Review by patosullivan — APPROVED** [review]
Looks good on device (other than the weird animation issue that is difficult to repro, maybe just a debug build issue)

**Pull request merged**